jenkins:  Friesian feature tests AssertionError

def test_clip(self):
        file_path = os.path.join(self.resource_path, "friesian/feature/parquet/data1.parquet")
        feature_tbl = FeatureTable.read_parquet(file_path)
        clip_tbl = feature_tbl.clip(["col_1", "col_2", "col_3"], min=2, max=None)
        assert isinstance(clip_tbl, FeatureTable), "clip_tbl should be a FeatureTable"
        assert feature_tbl.df.filter("col_1 < 2").count() != 0 and feature_tbl \
            .df.filter("col_2 < 2").count() != 0, "feature_tbl should not be changed"
        assert clip_tbl.df.filter("col_1 < 2").count() == 0, "col_1 should >= 2"
        assert clip_tbl.df.filter("col_2 < 2").count() == 0, "col_2 should >= 2"
        assert clip_tbl.df.filter("col_3 < 2").count() == 0, "col_3 should >= 2"
        with self.assertRaises(Exception) as context:
            feature_tbl.clip(None, 2)
        self.assertTrue('columns should be str or list of str, but got None.'
>                       in str(context.exception))
E       AssertionError: False is not true
../test/zoo/friesian/feature/test_table.py:224: AssertionError

jenkins link:
http://10.239.176.111:18888/job/ZOO-PR-Python-Spark-2.4-py36/4437/console

--- a/pyzoo/test/zoo/friesian/feature/test_table.py
+++ b/pyzoo/test/zoo/friesian/feature/test_table.py
@@ -220,7 +220,7 @@ class TestTable(TestCase):
         assert clip_tbl.df.filter("col_3 < 2").count() == 0, "col_3 should >= 2"
         with self.assertRaises(Exception) as context:
             feature_tbl.clip(None, 2)
-        self.assertTrue('columns should be str or list of str, but got None.'
+        self.assertTrue('columns should be str or a list of str, but got None.'
                         in str(context.exception))
 
         feature_tbl = FeatureTable.read_parquet(file_path)

